tools/mcmanifest.js

```diff
@@ -2068,9 +2070,11 @@ export class Tool extends TOOL {
 		}
 		return;
 	}
-	mergeProperties(targets, sources) {
+	mergeProperties(targets, sources, exclude) {
 		if (sources) {
 			for (let name in sources) {
+				if (exclude && exclude.includes(name))
```

Comment:
tiny nit: this could be simplified to `if (exclude?.includes(name))`

Reply:
> tiny nit: this could be simplified to `if (exclude?.includes(name))`

no problem - that's the right way to do it.  change made.